docs/source/jit.md

```diff
@@ -0,0 +1,880 @@
+# TorchScript
+
+```{toctree}
+:maxdepth: 1
+:caption: Builtin Functions
+:hidden:
+
+torch.jit.supported_ops <jit_builtin_functions>
+```
+
+```{toctree}
```

Comment:
where did [this](https://github.com/pytorch/pytorch/blob/main/docs/source/jit.rst?plain=1#L12-L17) go?